app/styles/tailwind.css

```
@@ -213,7 +220,15 @@
 			margin-bottom: 0px;
 		}
 	}
-
+	.ellipsis {
+		position: relative;
+		font-size: 24px;
+		padding-left: 0px;
+		line-height: 0.75em;
```

This change makes the ellipsis look more like it belongs to the text (same font size).

```suggestion
		padding-left: 0px;
```